src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/PXActionExecution/PXActionExecutionInPXGraphInitializationTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.PXActionExecution;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.PXActionExecution
+{
+    public class PXActionExecutionInPXGraphInitializationTests : DiagnosticVerifier
+    {
+        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>
+            new PXGraphAnalyzer(new PXActionExecutionInPXGraphInitializerAnalyzer());
+
+        [Theory]
+        [EmbeddedFileData(@"PXGraph\PressInGraph.cs")]
+        public Task Press_PXGraphInitialization(string actual) =>
```

Comment:
Please use `async` / `await` because if the task is returned directly, any exceptions thrown during the test execution will be wrapped in .NET system exception which decreases test output readability.

Reply:
ok